pkg/controller/tenant/tenant_controller.go

```diff
@@ -127,6 +128,7 @@ func (tc *TenantController) Run(workers int, stopCh <-chan struct{}) {
 	defer tc.queue.ShutDown()
 
 	klog.Infof("Starting tenant controller.")
+	tc.cerateSystemTenantIfNotExist()
```

Comment:
if you have time, please add a test in the tenant controller integration test here:

https://github.com/futurewei-cloud/arktos/blob/master/test/integration/tenantcontroller/tenantcontroller_test.go

Reply:
Added test case TestSystemTenantCreatedAutomically as suggested